src/main/proto/gtfs-realtime.proto

```diff
@@ -843,6 +843,26 @@ message VehicleDescriptor {
   // The license plate of the vehicle.
   optional string license_plate = 3;
 
+  enum WheelchairAccessible {
```

Comment:
This will make it harder to update to the latest version of the published gtfs realtime proto file.

What do you think about using an protobuf extension instead? Would that work here?

I've done that in a downstream repo: https://github.com/mfdz/OpenTripPlanner/commit/6eb3b42377825390373ae3399dd1ad6af1009c21

Reply:
More examples how to use it: https://github.com/mfdz/OpenTripPlanner/blob/87e462884ec37d17cf12c3439adc68c95d169606/src/main/java/org/opentripplanner/updater/stoptime/MfdzStopTimeExtension.java#L14